### DISCUSSION TITLE
blog/2021/08/25/twfe-diagnostics/index

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#57] [Blog comments]
# blog/2021/08/25/twfe-diagnostics/index

Use R to explore possible biases that come from differential treatment timing in two-way fixed effects (TWFE) regression models

https://www.andrewheiss.com/blog/2021/08/25/twfe-diagnostics/index.html

**Comment by IAFomichev** [comment]
Sir, you may one day be canonized.

**Comment by polkaen** [comment]
THANK YOU!! 🙏 It is so nice to not only have methods broken down and explained, but going through R code and the output is 💯. Much Appreciated as I continue on through this TWFE journey.
